react/features/base/config/reducer.js

```diff
@@ -194,6 +194,28 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.toolbarButtons = interfaceConfig.TOOLBAR_BUTTONS;
     }
 
+    newValue.disabledSounds = newValue.disabledSounds || [];
+
+    if (oldValue.disableJoinLeaveSounds) {
+        newValue.disabledSounds = newValue.disabledSounds
```

Comment:
This can now be written as: `newValue.disabledSounds.unshift(...[ 'PARTICIPANT_LEFT_SOUND', 'PARTICIPANT_JOINED_SOUND' ]);`

No strong opinion, but it does feel a bit more concise?